### PR TITLE
Updated content structure and submission guide

### PR DESCRIPTION
**Pull request by jmenglund**
I have made the following changes after discussion with @mattiasstromberg: 

* Simplified navigation with 4 menu items to better align with how the information is presented at Central EGA (Submit Data, Access Data, Privacy Notice and Contact)
* Removed unused files: `submission.md` and `request.md`
* Renamed files: `legal.md` => `privacy-notice.md`, `request-dataset.md`=> `data-access.md`
* Updated text on privacy notice page
* Removed `about.md` since the information will be included in the front page
*  Added page about how to assess if SDA is suitable (`is-sda-suitable.md`)
* Updated submission guide text
* Fixed hight issue for div elements in submission guide
* Updated text on the front page

### CODE REVIEW

File: contact.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+layout: article
+titles:
+  # @start locale config
+  en      : &EN       Contact us
+  # @end locale config
+key: page-single
+---
+
+If you have questions or comments, please write to the
+<a href="mailto:">SDA Helpdesk</a>.
```
Comment (norling) [inline contact.md]: Even if we don't have a correct helpdesk email, we should add _some_ address here I think, so it's not forgotten.

File: data-access.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    en      : &EN       Accessing Data in SDA
+    # @end locale config
+key: page-single
+---
+
+This page contains the information needed in order to request access to data
+submitted to SDA.
+
+## How to request a dataset
+- Apply for a dataset
```
Comment (norling) [inline data-access.md]: Can we add more information here? as a researcher I would have no idea what to do!

File: _includes/quick-guide-table.html [hunk 3]
```diff
@@ -51,24 +51,22 @@
...
-    <div >Encrypt your data files using the the instructions found <a href="https://github.com/NBISweden/EGA-SE-user-docs">here.</a> </div>
-    <div class="steps">6. Register experiments and runs <div class="arrow arrow-down"></div> </div>
-    <div >Associate each data file to a registered sample and study by Linking files to samples . Details of the experimental procedure you followed must be provided.</div>
-    <div class="steps">7. Finish your submission <div class="arrow arrow-down"></div> </div>
-    <div >Group your runs/analysis into datasets and link them to your new or existing DAC and policy using Submitter Portal 
-        or an XML based programmatic submission. Data request are done at a dataset level, thus files within a datasets must share release conditions.</div>
-    <div class="steps">8. Release </div>
-    <div>Instruct our Helpdesk to release your study. 
-        All registered studies are automatically placed on hold until the named submission or DAC contact instructs our Helpdesk for the study to be released</div>
+  <div class="steps">1. Is SDA the right archive? <div class="arrow arrow-down"></div></div>
```
Comment (norling) [inline _includes/quick-guide-table.html]: I would prefer to have these lines split at 80 characters to make it more readable without the markdown parser. This is not important though, if you disagree! :)

File: sda-submission-guide.md [hunk 4]
```diff
@@ -4,7 +4,9 @@ title: SDA Submission Guide
 key: page-single
 ---
 
-This is a guide to submit data to SDA.
+This guide describes how to submit data to the Swedish Sensitive Data
+Archive (SDA). If you have questions or comments, please write to the
+<a href="mailto:">SDA Helpdesk</a>.
```
Comment (norling) [inline sda-submission-guide.md]: Again - we should add _some_ email address...

File: is-sda-suitable.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    # @start locale config
+    en      : &EN       Is SDA suitable for my data?
+    # @end locale config
+key: page-single
+---
+
+Before you start working on a submission to SDA, you should assess whether the
+repository is suitable or not. This guide will help you to make this assessment.
+
+## Are the data of the right type?
```
Comment (norling) [inline is-sda-suitable.md]: grammar:
```suggestion
## Is the data of the right type?
```

**Review by norling — APPROVED** [review]
Had some small comments, but looks really good in general! :)

File: _includes/quick-guide-table.html [hunk 6]
```diff
@@ -51,24 +51,22 @@
...
-    <div >Associate each data file to a registered sample and study by Linking files to samples . Details of the experimental procedure you followed must be provided.</div>
-    <div class="steps">7. Finish your submission <div class="arrow arrow-down"></div> </div>
-    <div >Group your runs/analysis into datasets and link them to your new or existing DAC and policy using Submitter Portal 
-        or an XML based programmatic submission. Data request are done at a dataset level, thus files within a datasets must share release conditions.</div>
-    <div class="steps">8. Release </div>
-    <div>Instruct our Helpdesk to release your study. 
-        All registered studies are automatically placed on hold until the named submission or DAC contact instructs our Helpdesk for the study to be released</div>
+  <div class="steps">1. Is SDA the right archive? <div class="arrow arrow-down"></div></div> 
+  <div>Assess whether SDA is the right place for your data. The SDA can only take certain types of data and the data submitted to the repository must require controlled access. Read about how to <a href="../is-sda-suitable.html">find out if SDA is suitable or not</a>.</div>
```
Comment (aaperis) [inline _includes/quick-guide-table.html]: Maybe include a link to where controlled access is explained in more detail. Like e.g. `is-sda-suitable.md#should-your-submission-require-controlled-access?`

**Review by dbampalikis — APPROVED** [review]
Pair reviewed with @aaperis.Apart for the minor comments, it looks good and it improves the quality of the info. Good job!

**Review by aaperis — APPROVED** [review]
Looks good, one comment that may be worth considering

File: contact.md (hunk 1 above)
Comment (jmenglund) [reply contact.md]: Yes I agree. I'll check which one we can use.

File: data-access.md (hunk 2 above)
Comment (jmenglund) [reply data-access.md]: Yes, I think I will be add at least something.

File: _includes/quick-guide-table.html (hunk 3 above)
Comment (jmenglund) [reply _includes/quick-guide-table.html]: I agree

File: sda-submission-guide.md (hunk 4 above)
Comment (jmenglund) [reply sda-submission-guide.md]: Agree

File: _includes/quick-guide-table.html (hunk 6 above)
Comment (jmenglund) [reply _includes/quick-guide-table.html]: I think the link to the page is sufficient, but will explain "controlled access" in the text.

File: data-access.md (hunk 2 above)
Comment (jmenglund) [reply data-access.md]: I have added some new text

File: sda-submission-guide.md (hunk 4 above)
Comment (jmenglund) [reply sda-submission-guide.md]: Fixed

File: contact.md (hunk 1 above)
Comment (jmenglund) [reply contact.md]: Fixed

**Review by norling — APPROVED** [review]
Looks great! :)

**Pull request merged**